docs/pages/product/apis-integrations/mdx-api.mdx

```diff
@@ -0,0 +1,76 @@
+# MDX API
+
+The MDX (MultiDimensional eXpressions) API provides a robust interface for connecting desktop applications, such as Microsoft Excel, to the Cube Cloud semantic layer via the XMLA (XML for Analysis) standard. 
```

Comment:
- Do we know any application, other than Microsoft Excel, that can be used with the MDX API? If we do, let's name it. If we don't, let's remove the mention of it. As a rule of thumb, it's great to be more specific than vague in the docs.
- Can we add links to some neutral/authoritative pages for MDX and XMLA? Wikipedia or some Microsoft resources would be great.

Reply:
It would also be great, just like on REST API and SQL API pages, to reference a use case, explain query format, show an example request. Also, since Microsoft Excel is mentioned here, and it was never supported before, it would be great to add a page for Microsoft Excel, similarly to what we already have for [Power BI](https://cube.dev/docs/product/configuration/visualization-tools/powerbi).